pages/resultsPage.py

```diff
@@ -0,0 +1,59 @@
+from datetime import date,timedelta
+
+class connection():
+    def __init__(self, stations,date,deparr,duration,transfers):
```

Comment:
Dobrze jest mieć pod ręką zainstalowany autoformater kodu, PyCharm chyba taki ma z pudełka. Kwestia pamiętania aby od czasu do czasu wciśnąć skrót klawiszowy/ikonkę.
A najlepiej mieć czuwającego [lintera](https://pypi.org/project/pylint/), ale to już wymaga nieco wysiłku i konfigurację.
```suggestion
    def __init__(self, stations, date, deparr, duration, transfers):
```